src/main/java/com/volunteer/enterprise/dao/VolunteerDAOStub.java

```diff
@@ -11,10 +10,9 @@
 
 
 @Repository
-public class VolunteerServiceDAOStub implements IVolunteerServiceDAO {
+public class VolunteerDAOStub implements IVolunteerDAO {
```

Comment:
Renamed `VolunteerServiceDAO` to more accurate `VolunteerDAO`.

Reply:
Right... it wouldn't be both.  :)